### ISSUE TITLE
Tauri "Uploading the artifacts to a release" failing even though it worked before

### ISSUE THREAD

**Issue by p32929** [tauri-apps/tauri-action#259]
Hello, I've used this code from this repo:
```
name: "release"
on:
    pull_request:
        branches:
            - master

env:
  CI: false

jobs:
    create-release:
        runs-on: ubuntu-latest
        outputs:
            RELEASE_UPLOAD_ID: ${{ steps.create_release.outputs.id }}

        steps:
            - uses: actions/checkout@v2
            - name: setup node
              uses: actions/setup-node@v1
              with:
                  node-version: 14
            - name: get version
              run: echo "PACKAGE_VERSION=$(node -p "require('./package.json').version")" >> $GITHUB_ENV
            - name: create release
              id: create_release
              uses: actions/create-release@v1.1.0
              env:
                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
              with:
                  tag_name: ${{ env.PACKAGE_VERSION }}
                  release_name: "Remix v${{ env.PACKAGE_VERSION }}"
                  body: CHANGELOG.md
                  draft: false
                  prerelease: false
    build-tauri:
        needs: create-release
        strategy:
            fail-fast: false
            matrix:
                platform: [macos-latest, ubuntu-latest, windows-latest]

        runs-on: ${{ matrix.platform }}
        steps:
            - uses: actions/checkout@v2
            - name: setup node
              uses: actions/setup-node@v1
              with:
                  node-version: 14
            - name: install Rust stable
              uses: actions-rs/toolchain@v1
              with:
                  toolchain: stable
            - name: install tauri bundler
              run: cargo install tauri-bundler --force
            - name: install webkit2gtk libgtk-3-dev libappindicator3-dev (ubuntu only)
              if: matrix.platform == 'ubuntu-latest'
              run: |
                  sudo apt-get update
                  sudo apt-get install -y webkit2gtk-4.0 libgtk-3-dev libappindicator3-dev
            - name: install app dependencies and build it
              run: npm install && npm run build
            - uses: tauri-apps/tauri-action@v0
              env:
                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
              with:
                  releaseId: ${{ needs.create-release.outputs.RELEASE_UPLOAD_ID }}
```

( I have only added `CI: false` as it was failing otherwise )
& it worked before. But nor sure what happened later, it keeps on showing these errors:

![Screenshot (6)](https://user-images.githubusercontent.com/6418354/164619366-4bd9d438-0531-4e8d-944b-800b5b010eb0.png)
![Screenshot (7)](https://user-images.githubusercontent.com/6418354/164619373-ce9ea1d9-a51b-4bbc-8b6b-9c28fdfd88f9.png)

I'm not sure if the error is from github action itself or the code. 

Thanks

**Comment by FabianLars** [comment]
That looks like a github issue because it didn't run _anything_, not even the first step (the repo checkout)? So idk, I'd suggest trying it again in a few hours.

**Comment by p32929** [comment]
Tried again but still failing. Tbh, I've been trying in a few hours for like 3 days now over and over again, tried changing so many things, but still fails :(

**Comment by FabianLars** [comment]
The problem is that it's the other job that (seemingly) fails, not the tauri specific one (if the screenshots you provided are accurated and there is no other information _somewhere_). So i don't really know how we're supposed to help here.

But since the release actions is unmaintained for such a long time now, i could imagine that it's just straight up broken by now. We probably should remove this example from our readme.

**Comment by p32929** [comment]
If you want, I can actually share the source code of the app with you ( Give you access to the code in github ). The code is not that clean, so I was just embarrassed to make it open source.

**Comment by FabianLars** [comment]
The code itself doesn't matter according to your screenshots, since it doesn't even get to building the tauri app. So only the logs of the workflow runs are interesting, but again, you already showed them.
But sure, feel free to add me to your repo, maybe i see something interesting.

Other than that i'll look into removing this specific example from the readme. And to "fix" your issue you can try letting tauri-action do the release publishing (or search for another action than what our example uses if you need a seperate publishing job)

**Comment by p32929** [comment]
Added you into the repo :)

& I'm an absolute beginner in github actions tbh and this is my first time doing github actions and I was really surprised when the first few times it worked tbh. 

I guess, I'll have to look into some other open source tauri apps repo.

**Comment by p32929** [comment]
Just tried all the actions found in thes apps from here: https://github.com/tauri-apps/awesome-tauri/pulls
& All failed

![Screenshot 2022-04-23 at 21-38-50 Actions · p32929_notepadta](https://user-images.githubusercontent.com/6418354/164913024-8c58d306-5a09-4807-8410-be7c9ba886f6.png)

Now, I don't even know what to do :/

**Comment by FabianLars** [comment]
I opened an issue at your repo.
tldr: limited ci build minutes in private repos, how did I forget that one...
